README.md

```diff
@@ -17,38 +23,216 @@ npm i @dialonce/boot
 Require it as the first instruction (after env vars are set)
 ```js
 require('@dialonce/boot')({
-    LOGS_TOKEN: '',
-    BUGS_TOKEN: ''
+    LOGS_TOKEN: '', // logentries token
+    BUGS_TOKEN: '', // bugsnag token
+    MIN_LOG_LEVEL: '',  // minimal log level to process a message
+    DEFAULT_LOG_LEVEL: '', // default log level for message if not given
+    MIN_LOG_LEVEL_CONSOLE: '', // minimal log level to process a message by console logger chain link
+    MIN_LOG_LEVEL_LOGENTRIES: '', // minimal log level to process a message by logentries logger chain link
+    MIN_LOG_LEVEL_BUGSNAG: '', // minimal log level to process a message by a bugsnag logger chain link
```

Comment:
I don't think we need this because we always just notify with bugsnag on errors.